zookeeper-docs/src/main/resources/markdown/zookeeperAdmin.md

```diff
@@ -1739,6 +1739,12 @@ and [SASL authentication for ZooKeeper](https://cwiki.apache.org/confluence/disp
     Disabling it only recommended for testing purposes.
     Default: true
 
+* *ssl.clientHostnameVerification* and *ssl.quorum.clientHostnameVerification* :
+    (Java system properties: **zookeeper.ssl.clientHostnameVerification** and **zookeeper.ssl.quorum.clientHostnameVerification**)
+    **New in (INSERT VERSION HERE):**
```

Comment:
3.9.3

Reply:
689794749d71489c849fcaf8137a98c39dbd250a

Skipped 3.9.3 since it's already released at this point